CRTX-258891-AWS - Add Network Firewall subnet association commands (#44948)
demisto/content: Packs/AWS/ReleaseNotes/2_4_2.md

#### Integrations

##### Amazon Web Services

> - Added support for **aws-network-firewall-subnet-change-protection-update** command that modifies the flag, SubnetChangeProtection, which indicates whether it is possible to change the subnets that the firewall is associated with. required permissions: network-firewall:UpdateSubnetChangeProtection.

Reviewer: ```suggestion
- Added support for the ***aws-network-firewall-subnet-change-protection-update*** command, which modifies the *SubnetChangeProtection* flag to indicate whether subnets associated with the firewall can be changed. Required permissions: network-firewall:UpdateSubnetChangeProtection.
```

> - Added support for **aws-network-firewall-subnets-disassociate** command that removes the specified subnet associations from the firewall. This removes the firewall endpoints from the subnets and removes any network filtering protections that the endpoints were providing. required permissions: network-firewall:DisassociateSubnets.

Reviewer: ```suggestion
- Added support for the ***aws-network-firewall-subnets-disassociate*** command, which disassociates specified subnets from the firewall. This removes firewall endpoints from the subnets, revoking any network filtering protections they provided. Required permissions: network-firewall:DisassociateSubnets.
```

> - Added support for **aws-network-firewall-subnets-associate** command that associates the specified subnets in the Amazon VPC to the firewall. You can specify one subnet for each of the Availability Zones that the VPC spans. This is used to provide the firewall endpoints for the subnets. required permissions: network-firewall:AssociateSubnets.

Reviewer: ```suggestion
- Added support for the ***aws-network-firewall-subnets-associate*** command, which associates the specified subnets in the Amazon VPC to the firewall. You can specify one subnet for each of the Availability Zones that the VPC spans. This is used to provide the firewall endpoints for the subnets. Required permissions: network-firewall:AssociateSubnets.
```
